lib/Net/Async/Github.pm

```diff
@@ -211,6 +211,39 @@ sub pr {
     )
 }
 
+=head2 prs
+
+Returns information of all PRs of given repository.
```

Comment:
information is too general, the return is some objects  or hash, or some type.
and is it going to filter by only `open` PR?

Reply:
yes it only return the opening PR.